src/components/Requirements/RequirementHeader.vue

```diff
@@ -8,7 +8,27 @@
       {{ req.groupName }} Requirements
     </h1>
     <!-- TODO change for multiple colleges -->
-    <div v-if="reqIndex == numOfColleges" class="major">
+    <div v-if="reqIndex === 0" class="college">
+      <button
+        :style="{
+          'border-bottom': `2px solid #${reqGroupColorMap[req.groupName][0]}`,
+        }"
+        class="college-title-button college-title"
+        :disabled="true"
+      >
+        <p
+          :style="{
+            'font-weight': '500',
+            color: `#${reqGroupColorMap[req.groupName][0]}`,
+          }"
+          class="college-title-top"
+        >
+          {{ getCollegeFullName(onboardingData.college) }}
+        </p>
+      </button>
+    </div>
+    <!-- TODO change for multiple colleges -->
```

Comment:
this comment is copied over twice. remove the one above

Reply:
I purposefully copied it over because the indexing in both `v-if`s would be broken if we ever implemented multiple colleges. Do you still think it should be removed?

Reply:
Ok then i think that's fine.